src/Routers/FilesRouter.js

```diff
@@ -67,7 +66,7 @@ export class FilesRouter {
     return router;
   }
 
-  getHandler(req, res) {
+  async getHandler(req, res) {
```

Comment:
Can this just be changed to async without any other changes in the calling code?

Reply:
yes the getHandler is just used in an express route, and express don't care about async or sync callbacks